GOSSIP.md

```diff
@@ -2,77 +2,323 @@
 
 As explained in the [Sync](SYNC.md) document, any blockchain implementation
 needs communication with peers in order to be truly useful. Apart from the lower
-level details described in that document, there is the additional dimension of
+level details described in that document, there is the added dimension of
 how peers are selected for communication, how connections to those peers are
 maintained over time. This is the purpose of gossiping functionality.
 
 ## Initial Configuration
 
-The initial set of peers is defined from configuration when starting the node.
+The initial set of peers is defined by configuration when starting the node.
 This is a predefined set of peers that are automatically connected to upon
 startup.
 
 ## Peers
 
-A peer is identified by a URI consisting of the protocol `aenode://`, the public
-key, an `@` character, the hostname or IP number, a `:` character and the Noise
-port number the node is using. Example:
+A peer is identified by a URI consisting of the protocol 'aenode://', the public
+key, an '@' character, the hostname or IP number, a ':' character and the Noise
+port number the node is using. If the address contains a hostname, it will be
+resolved to an IP. Example:
 
-```
+'''
```

Comment:
These should still be backpacks, right? 🙂

Reply:
There are a few more places in the document that uses single quotes instead of backticks.